acajeremic/functions/homework.js

```diff
@@ -0,0 +1,55 @@
+/*Homework
+Create functions for all the learned arithmetic operations, and use them in the file, console.log results. Every function needs to accept 2 params.
++
+-
+/
+*
+**
+%
+
+*/
+let a;
+let b;
```

Comment:
You should not declare these variables here, they're not used anywhere, and might lead to some confusion in VSCode.
There is a difference between a variable and function parameter. the:
```
let a;
```
is not used in
```
function (a) {}
```
but instead that `a` from function becomes inner scope of function declaration, and `a` above is..,. well, above it, as a declared function.